### PR TITLE
[new release] linol (3 packages) (0.6)

### PR DESCRIPTION
**Pull request by c-cube**

### CODE REVIEW

**Comment by shonfeder** [comment]
Is the linting error here worth fixing?

```
2024-05-29 17:38.31: Exec: "git" "merge" "-q" "--" "0b9453aa6048fe22dfdf9af320fe1dd40296d350"
2024-05-29 17:38.33: Warning in linol-lwt.0.6: Dubious use of 'dune subst'. 'dune subst' should always only be called with {dev} (i.e. ["dune" "subst"] {dev}) If your opam file has been autogenerated by dune, you need to upgrade your dune-project to at least (lang dune 2.7).
2024-05-29 17:38.33: Warning in linol-eio.0.6: Dubious use of 'dune subst'. 'dune subst' should always only be called with {dev} (i.e. ["dune" "subst"] {dev}) If your opam file has been autogenerated by dune, you need to upgrade your dune-project to at least (lang dune 2.7).
2024-05-29 17:38.33: Warning in linol.0.6: Dubious use of 'dune subst'. 'dune subst' should always only be called with {dev} (i.e. ["dune" "subst"] {dev}) If your opam file has been autogenerated by dune, you need to upgrade your dune-project to at least (lang dune 2.7).
2024-05-29 17:38.46: Job failed: 3 errors:
Warning in linol-eio.0.6: Dubious use of 'dune subst'. 'dune subst' should always only be called with {dev} (i.e. ["dune" "subst"] {dev}) If your opam file has been autogenerated by dune, you need to upgrade your dune-project to at least (lang dune 2.7).
Warning in linol-lwt.0.6: Dubious use of 'dune subst'. 'dune subst' should always only be called with {dev} (i.e. ["dune" "subst"] {dev}) If your opam file has been autogenerated by dune, you need to upgrade your dune-project to at least (lang dune 2.7).
Warning in linol.0.6: Dubious use of 'dune subst'. 'dune subst' should always only be called with {dev} (i.e. ["dune" "subst"] {dev}) If your opam file has been autogenerated by dune, you need to upgrade your dune-project to at least (lang dune 2.7).
```

**Comment by c-cube** [comment]
nope

File: packages/linol-eio/linol-eio.0.6/opam [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+  "linol" {= version}
+  "base-unix"
+  "eio" {>= "1.0" & < "2.0"}
+  "eio_main" {with-test}
+  "lsp" {>= "1.17" & < "1.18"}
+  "jsonrpc" {>= "1.17" & < "1.18"}
+  "odoc" {with-doc}
+]
+build: [
+  ["dune" "subst"] {pinned}
```
Comment (mseri) [inline packages/linol-eio/linol-eio.0.6/opam:20]:
```suggestion
  ["dune" "subst"] {dev}
```

File: packages/linol-lwt/linol-lwt.0.6/opam [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+  "yojson" {>= "1.6"}
+  "linol" {= version}
+  "base-unix"
+  "lwt" {>= "5.1" & < "6.0"}
+  "lsp" {>= "1.17" & < "1.18"}
+  "jsonrpc" {>= "1.17" & < "1.18"}
+  "odoc" {with-doc}
+]
+build: [
+  ["dune" "subst"] {pinned}
```
Comment (mseri) [inline packages/linol-lwt/linol-lwt.0.6/opam:19]:
```suggestion
  ["dune" "subst"] {dev}
```

File: packages/linol/linol.0.6/opam [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+  "yojson" {>= "1.6"}
+  "logs"
+  "trace" {>= "0.4"}
+  "lsp" {>= "1.17" & < "1.18"}
+  "jsonrpc" {>= "1.17" & < "1.18"}
+  "ocaml" {>= "4.14"}
+  "odoc" {with-doc}
+]
+build: [
+  ["dune" "subst"] {pinned}
```
Comment (mseri) [inline packages/linol/linol.0.6/opam:19]:
```suggestion
  ["dune" "subst"] {dev}
```

**Comment by c-cube** [comment]
Thank you :)

**Pull request merged**